LoggingKata/Program.cs

```diff
@@ -19,11 +19,41 @@ static void Main(string[] args)
             logger.LogInfo($"Lines: {lines[0]}");
 
             var parser = new TacoParser();
+            ITrackable firstlocation = null;
+            ITrackable secondlocation = null;
+            double distance = 0;
 
             var locations = lines.Select(parser.Parse).ToArray();
 
-            // TODO:  Find the two Taco Bells in Alabama that are the furthest from one another.
-            // HINT:  You'll need two nested forloops
+            for (int i = 0; i < locations.Length; i++)
+            {
+                double findFurthestdistance = 0;
```

Comment:
This is an odd name for a variable with the verb "find".  Is there a better name potentially? Also, it should probably be created closer to where it's used, such as line 41